pages/1.13/security/ent/iam-api/index.md

```diff
@@ -41,14 +41,50 @@ All IAM endpoints require an authentication token and the `dcos:superuser` permi
 
 ### Via the IAM API
 
-To get an authentication token, pass a user name and password in the body of a request to `/auth/login`. It returns an authentication token as shown below.
+To get an authentication token, pass the credentials of a local user or service accout in the body of a `POST` request to `/auth/login`.
+
+To log in local user accounts supply `uid` and `password` in the request.
+
+**NOTE**: Read how to [establish trust in curl commands](https://docs.mesosphere.com/1.13/security/ent/tls-ssl/ca-trust-curl/) with DC/OS.
```

Comment:
Use the Note format listed on https://wiki.mesosphere.com/display/DOCS/Documentation+Style+Guide#DocumentationStyleGuide-Notes

Reply:
https://github.com/mesosphere/dcos-docs-site/pull/2007/files#diff-59add86963e731df499da35b41d2efd1L42
https://github.com/mesosphere/dcos-docs-site/pull/2007/files#diff-59add86963e731df499da35b41d2efd1R88

Can we reword `Via` to `Using` in the above two lines?